client/my-sites/plugins-wpcom/plugin-panel.jsx

```diff
@@ -24,14 +37,15 @@ export const PluginPanel = React.createClass( {
 				<Card className="wpcom-plugin-panel__panel-footer" href={ standardPluginsLink }>
 					{ this.translate( 'View all standard plugins' ) }
 				</Card>
-				<PremiumPluginsPanel />
-				<BusinessPluginsPanel />
+				<PremiumPluginsPanel isActive={ hasPremium } />
+				<BusinessPluginsPanel isActive={ hasBusiness } />
 			</div>
 		);
 	}
 } );
 
 const mapStateToProps = state => ( {
+	plan: getSelectedSite( state ).plan,
```

Comment:
Good point - it doesn't appear to be a problem here because the plugin-panel only loads if a specific site is already given. 

I have added a defaulting getter so that we don't risk throwing an exception.